ranger/ext/clipboard_util.py

```diff
@@ -0,0 +1,39 @@
+# This file is part of ranger, the console file manager.
+# License: GNU GPL version 3, see the file "AUTHORS" for details.
+
+
+def clipboards():
+    """Return a list of commands that can be executed by subprocess.Popen
+to copy some content into the system clipboard and proamry X
+selection. The spawn processes will read the contents from the
+standard input.
+
+    >>> new_clipboard_contents = 'some string'
```

Comment:
This looks like a doctest that doesn't really test things work? Could you check whether the clipboard has the right contents afterwards? Conditional on one of the clipboard utilities being available of course.